draft-ietf-add-ddr.md

```diff
@@ -171,44 +171,50 @@ Encrypted Resolver name.
 
 ## Authenticated Discovery {#authenticated}
 
+If the Unencrypted Resolver's IP address is in a public range, it is considered
```

Comment:
This needs a strong definition for "public range", citing the v6 addressing architecture or whatever else applies.